latex/chapter/Stand_der_Technik.tex

```diff
@@ -1,40 +1,46 @@
 
-Neben den Hauptfeatures MFCC und LPC wurden in dem Buch auch noch Gammatone Features, welche das menschliche Hören modellieren sollen, bewertet.
+Neben den Hauptmerkmalen MFCC und LPC wurden in dem Buch auch Gammatone-Merkmale bewertet, die das menschliche Hören modellieren sollen.
 Diese können auch relevante Daten für eine Sprecherauthentifizierung liefer \autocite[vgl.][S. 111, 117]{li_speaker_2012}.
 
-Qi (Peter) Li hat im Zuge seiner Arbeit auch noch ein ganz neues Merkmal (CFCC) entwickelt, das ebenfalls das menschliche Gehör nachahmen soll und in seinen Untersuchungen am besten abgeschnitten hat \autocite[vgl.][S. 135]{li_speaker_2012}.
+Qi (Peter) Li hat im Zuge seiner Arbeit auch ein ganz neues Merkmal (CFCC) entwickelt, das ebenfalls das menschliche Gehör nachahmen soll und in seinen Untersuchungen am besten abgeschnitten hat \autocite[vgl.][S. 135]{li_speaker_2012}.
 
-Das Buch \gqq{Advanced Topics In Biometrics} von Haizhou Li, Kar-ann Toh und Liyuan Li nennt MFCC das relevanteste Feature für die Sprecherauthentifizierung, auch wenn das Merkmal uhrsprünglich für die Spracherkennung entwickelt wurde \autocite[vgl.][S. 7, 51]{li_advanced_nodate}.
-Aber auch LPCC, PLP und FM wurden als effektiv bezeichnet und können zusammen mit MFCC sehr gute Ergebnisse erziehlen \autocite[vgl.][S. 6, 67]{li_advanced_nodate}.
+Das Buch \gqq{Advanced Topics In Biometrics} von Haizhou Li, Kar-ann Toh und Liyuan Li bezeichnet MFCC als das relevanteste Merkmal für die Sprecherauthentifizierung, obwohl es ursprünglich für die Spracherkennung entwickelt wurde \autocite[vgl.][S. 7, 51]{li_advanced_nodate}.
+Aber auch LPCC, PLP und FM werden als effektive Merkmale genannt und können zusammen mit MFCC sehr gute Ergebnisse erzielen \autocite[vgl.][S. 6, 67]{li_advanced_nodate}.
 
-Das Paper \gqq{Multilingual Speaker Recognition Using Neural Network} hatte mit der Extraktion von LPC und der Nutzung eines Neuronalen Netzes gute Ergebnisse erziehlt \autocite[vgl.][S. 9]{kumar_rajeev_multilingual_nodate}.
+Das Paper \gqq{Multilingual Speaker Recognition Using Neural Network} protokoliert gute Ergebnisse durch die Extraktion von LPC und der Nutzung eines neuronalen Netzes \autocite[vgl.][S. 9]{kumar_rajeev_multilingual_nodate}.
 
 Auch eine ältere Untersuchung von Reynolds and Rose ergab, dass LPC in einer ruhigen Umgebung effektiv ist, MFCC allerdings robuster ist und daher von den Autoren bevorzugt wurde \autocite[vgl.][S. 2f]{reynolds_robust_1995}.
+Als Klassifikations-Model wurden hierbei Gaussian Mixture Models (GMM) genutzt und als effektiv erachtet.
+Allerdings werden auch neuronale Netze, als ein neueres Modell, aufgeführt und auch Hidden Markov Models, welche eigentlich für Spracherkennung genutzt werden, werden als Möglichkeit erwähnt \autocite[vgl.][S. 2f, 11]{reynolds_robust_1995}.
```

Comment:
warum "als ein neueres Modell"? Ist das inhaltlich relevant?
Was heist "aufgeführt"? Nur genannt oder auch getestet?
Insgesamt ist der Satz finde ich auch zu verschachtelt/komplex -> evtl. aufteilen in zwei Sätze.